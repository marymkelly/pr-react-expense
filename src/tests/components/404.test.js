import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../components/pages/404.js';

test('should render 404 corectly', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(toJSON(wrapper)).toMatchSnapshot();
})

