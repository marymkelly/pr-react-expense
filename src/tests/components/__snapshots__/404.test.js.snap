// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 corectly 1`] = `
<div>
  404! - 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
